### DISCUSSION TITLE
suggestion: intergration with wikidata

### DISCUSSION THREAD

**Discussion by backache** [openbenches/openbenches.org#323] [Ideas]
I was thinking that as wikidata is such a trove of useful information, integrating with it at some level could be useful

For example, allowing a wikidata id to be attached to a record as a tag, for example, I live in a suburb of South London called Hayes and the benches mention Hayes, so you could add the wikidata id for it Q2690370

With that id you could show other records that have the same tag, or link to something like "reasonator" that  takes the ID does a query and shows all it knows (about Hayes for example) in a nice human-readable format https://reasonator.toolforge.org/?&q=2690370

My thinking is to encourage people to dig down in your data and also to answer questions people may have such as a placename or persons name (if they are notable enough to have a wikidata entry)

**Comment by edent** [comment]
That's an interesting idea. We already have metatags for location.

```
<meta property="place:location:latitude"  content="52.477772">
<meta property="place:location:longitude" content="-1.898581">
<meta itemprop="latitude"  content="52.477772" />
<meta itemprop="longitude" content="-1.898581" />
```

Can you suggest a good way to get a Wikidata ID from the data? And, I guess more importantly, how to expose them to a user?

Are there any example websites which do this well?

**Comment by backache** [comment]
I was thinking more of a tagcloud instead of microdata (though I sure there are things that can be done with that too)

**Comment by edent** [comment]
OK, I can look into it. Do you have any example sites I can copy from?

**Comment by jrbray1** [comment]
I use wikidata a lot, and don't think there is much useful integration. The context for a bench is very unclear, is is the churchyard, village, parish, town, county or country. All have wikidata items, but you'd be better off tying into OpenStreetMap, where they have the granularity of benches in their data.

**Comment by backache** [comment]
Well an extreme example of a bench that could do with more data linking to/from it is the one for the band "Black Sabbath", tourists looking at the benches might wonder who the people on the bench are and want to find out more, wikidata could give links to articles, pictures and of course music.

In some ways, openbenches could make the branch interactive!

https://openbenches.org/bench/24599

**Comment by jrbray1** [comment]
That would be great in a bench comment, but benches seem to be very rarely for famous people. I guess they aspire for statues...

**Comment by edent** [comment]
I really like the idea of adding Wikidata about the *subject* of a bench. But, as jbray1 says - most of them are for people without Wikipedia bios.

I wonder about doing the reverse though? Would it be possible to put the bench link into the Wikidata page of a subject?

**Comment by jrbray1** [comment]
The two ways of adding external links to wikidata are to propose a new property, where you formalise the URL, or to just add a bald 'described by source' OpenBenches with qualifier URL where you put the link

The former needs a consultation process, and it would probably be rejected as not notable enough. Anyone can add the latter once they have a wikidata account, but TBH they are in danger of deletion, as not being notable in most cases.

**Comment by backache** [comment]
OB isn't probably isn't big enough (or easy enough for its admins to get their heads around) for it to have its own "property" 
 (the equivalent in the triplet-world of a field)

They do have a property called URL that I have added OB too (but that may get removed by an admin) 

![image](https://user-images.githubusercontent.com/6274724/179519132-0f0b8a19-24a0-4a7c-b113-c884795e9a5f.png)

I have also added it in a property called "website account on" that may have more change of "sticking"

![image](https://user-images.githubusercontent.com/6274724/179519719-e5fc1c82-2bba-4a38-ba4d-7d49688cddbc.png)

**Comment by edent** [comment]
We have
https://www.wikidata.org/wiki/Q47526250
and
https://commons.wikimedia.org/wiki/Category:OpenBenches

So there are some Wiki[p|m]edia pages which use the photos - https://commons.wikimedia.org/wiki/Category:Benches_in_OpenBenches

I'm not immersed enough in Wiki-speak to know if that helps or hinders this idea :-)

**Comment by backache** [comment]
Nice, so a picture of an OB in a Wikipedia article (for example in Black Sabbaths) could lead (by following enough links) to OB itself.

![image](https://user-images.githubusercontent.com/6274724/179524616-015f678a-ca75-4d15-8cdb-1ba6f08571f5.png)
